specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -302,68 +308,68 @@
       ],
       "properties": {
         "name": {
-          "description": "The user-provided name of this container",
+          "description": "The user-provided name of the container instance.",
           "type": "string"
         },
         "properties": {
           "x-ms-client-flatten": true,
-          "description": "The container properties.",
+          "description": "The properties of the container instance.",
           "$ref": "#/definitions/ContainerProperties"
         }
       }
     },
     "ContainerProperties": {
-      "description": "The container properties.",
+      "description": "The container instance properties.",
       "type": "object",
       "required": [
         "image",
         "resources"
       ],
       "properties": {
         "image": {
-          "description": "The name of the image used when creating this container",
+          "description": "The name of the image used to create the container instance.",
           "type": "string"
         },
         "command": {
-          "description": "Command array to execute within the container in exec form.",
+          "description": "The commands to execute within the container instance in exec form.",
           "type": "array",
           "items": {
             "type": "string"
           }
         },
         "ports": {
-          "description": "The ports exposed by this container",
+          "description": "The exposed ports on the container instance.",
           "type": "array",
           "items": {
             "$ref": "#/definitions/ContainerPort"
           }
         },
         "environmentVariables": {
-          "description": "The environment variables to set in this container",
+          "description": "The environment variables to set in the container instance.",
           "type": "array",
           "items": {
             "$ref": "#/definitions/EnvironmentVariable"
           }
         },
         "instanceView": {
-          "description": "The instance view of the container. Only valid in response.",
+          "description": "The instance view of the container instance. Only valid in response.",
```

Comment:
Throughout all references to a single container, I updated it to state "container instance".